CalibTracker/SiStripChannelGain/test/PCL/step3_ALCA.py

```diff
@@ -20,25 +20,39 @@
 #process.load('Configuration.StandardSequences.Reconstruction_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
```

Comment:
This is not necessary anymore since 75X, can you please switch to `Configuration.StandardSequences.FrontierConditions_GlobalTag_cff`?